### ISSUE TITLE
Custom naming strategy

### ISSUE THREAD

**Issue by hnordt** [edgedb/edgedb-js#975]
The whole JS ecosystem has the camel case convention for property names.

It would be nice if this JS client supports the same naming strategy feature from the DotNet client https://docs.edgedb.com/libraries/dotnet/customtypes#using-a-naming-strategy, so the property names in query results could be camel case enforced.

**Comment by CarsonF** [comment]
Why not name your schema in camel case? that's what I do.

**Comment by CarsonF** [comment]
IMO that's the lesser evil, than trying to have naming translation for a specific client.
In practice we have very little snake case; pretty much just the builtin functions.

**Comment by hnordt** [comment]
I agree that a naming translation is bad, but it would be nice if we can do like Drizzle where you can "alias" the property name:

```js
{ myAge: integer('my_age') }
```

The SDL could support something like:

```
type Movie {
  required title: str;
  release_year as releaseYear: int64;
}
```
